### PR TITLE
Upgrade every package

### PR DESCRIPTION
**Pull request by rcantin-w**
## What does this change?

Everything but also seemingly nothing.
Noticed many [security flags](https://github.com/wellcomecollection/developers.wellcomecollection.org/security/dependabot?q=is%3Aopen+manifest%3Ayarn.lock+package%3Apath-to-regexp) and Dependabot PRs
I think no one actively looks after this so it makes sense, and as I'm in between tickets I went for a wild mass upgrade. 
Not much needed updating, I'll comment in-line.

I've also added a CODEOWNERS file for future PRs to get picked up.

## How to test

Run it locally (`yarn && yarn start`) and build (`yarn build`); is it happy?
But I'm not versed in this repo at all so will take any extra testing you might think about! Like I haven't run `swizzle` and `clear` (see `package.json`), but they're not in the README file and therefore am thinking might not be ever used?

## How can we measure success?

No security flags woo.

## Have we considered potential risks?
 Breaks the site and we fix it.

### CODE REVIEW

File: docs/examples/01-exploring-wellcome-collections-apis.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # 1. Exploring Wellcome Collection's APIs
 
-[View on GitHub](https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/main/notebooks/01-exploring-wellcome-collections-apis.ipynb) | [Run in Google Colab](https://colab.research.google.com/github/wellcomecollection/developers.wellcomecollection.org/blob/main/notebooks/01-exploring-wellcome-collections-apis.ipynb)
+[View on GitHub](https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/upgrade-everything/notebooks/01-exploring-wellcome-collections-apis.ipynb) | [Run in Google Colab](https://colab.research.google.com/github/wellcomecollection/developers.wellcomecollection.org/blob/upgrade-everything/notebooks/01-exploring-wellcome-collections-apis.ipynb)
```
Comment (rcantin-w) [inline docs/examples/01-exploring-wellcome-collections-apis.md:3]: I don't know why this happens? [Seems to be done automatically by wecobot](https://github.com/wellcomecollection/developers.wellcomecollection.org/pull/37/commits/ce345f0240e3f1a84550402a38f19ab7a857e9b2)? I'm guessing it's for internal nav and hopefully gets re-changed on merge?

File: docusaurus.config.js [hunk 2]
```diff
@@ -1,8 +1,6 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
-
-const lightCodeTheme = require("prism-react-renderer/themes/github");
-const darkCodeTheme = require("prism-react-renderer/themes/dracula");
+import { themes } from "prism-react-renderer";
```
Comment (rcantin-w) [inline docusaurus.config.js:3]: https://github.com/FormidableLabs/prism-react-renderer?tab=readme-ov-file#theming

File: docusaurus.config.js [hunk 3]
```diff
@@ -118,67 +116,67 @@ const config = {
...
         ],
-        copyright: `Except where otherwise noted, content on this site is licensed under a <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International Licence</a>.`
+        copyright: `Except where otherwise noted, content on this site is licensed under a <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International Licence</a>.`,
       },
       prism: {
-        theme: lightCodeTheme,
-        darkTheme: darkCodeTheme
-      }
+        theme: themes.vsLight,
+        darkTheme: themes.vsDark,
```
Comment (rcantin-w) [inline docusaurus.config.js:175]: https://github.com/FormidableLabs/prism-react-renderer?tab=readme-ov-file#theming

**Comment by rcantin-w** [comment]
Wecobot also overwrote this change https://github.com/wellcomecollection/developers.wellcomecollection.org/pull/37/commits/ce345f0240e3f1a84550402a38f19ab7a857e9b2#diff-25d9c047856e6fe3927f408e54d17c580d4e9a8b22c39ac99fb4280c18898601L5-R5

which is strange as [it's not a supported format](https://github.com/micromark/mdx-state-machine#723-autolinks), if you run it locally like that it'll error.

File: docs/examples/01-exploring-wellcome-collections-apis.md (hunk 1 above)
Comment (rcantin-w) [reply docs/examples/01-exploring-wellcome-collections-apis.md:3]: Robert confirmed it does. All good.

File: docs/examples/05-working-with-snapshots-of-the-api.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+[View on GitHub](https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/upgrade-everything/notebooks/05-working-with-snapshots-of-the-api.ipynb) | [Run in Google Colab](https://colab.research.google.com/github/wellcomecollection/developers.wellcomecollection.org/blob/upgrade-everything/notebooks/05-working-with-snapshots-of-the-api.ipynb)
 
-As we saw at the end of the last notebook, the API limits its responses to 10,000 total results - after that point, users are directed to work with the snapshots. For example, making a request to <https://api.wellcomecollection.org/catalogue/v2/works?pageSize=100&page=101> gives us:
+As we saw at the end of the last notebook, the API limits its responses to 10,000 total results - after that point, users are directed to work with the snapshots. For example, making a request to [https://api.wellcomecollection.org/catalogue/v2/works?pageSize=100&page=101](https://api.wellcomecollection.org/catalogue/v2/works?pageSize=100&page=101) gives us:
```
Comment (rcantin-w) [inline docs/examples/05-working-with-snapshots-of-the-api.md:5]: Autolinks are not supported https://github.com/micromark/mdx-state-machine#723-autolinks

**Review by kenoir — APPROVED** [review]
👏 Nice work.

**Pull request merged**